File: dataFeed/formatfeed.js
// const fs = require('fs');

// fs.readFile("./datafeed_kovan.txt", 'utf8', (err, data) => {
// 	console.log(data);
// })

let dataFeed = {};

var lineReader = require('readline').createInterface({
  input: require('fs').createReadStream('datafeed_kovan.txt')
});

lineReader.on('line', function (line) {
	line = line.replace("\t", " ").split(" ");
	dataFeed.kovan[line[2]] = line;
	console.log(dataFeed);
});

File: dataFeed/formatfeed.py
import json

dataFeed = {}

def keyNotExists(_network, _token):
	if not dataFeed.get(_network):
		dataFeed[_network] = {}
	if not dataFeed[_network].get(_token):
		dataFeed[_network][_token] = {}

with open('datafeed_kovan.txt', 'r') as reader:
	for line in reader.readlines():
		line = line.split()
		keyNotExists('kovan', line[2])
		dataFeed['kovan'][line[2]][line[0]] = line[3]
with open('datafeed_rinkeby.txt', 'r') as reader:
	for line in reader.readlines():
		line = line.split()
		keyNotExists('rinkeby', line[2])
		dataFeed['rinkeby'][line[2]][line[0]] = line[3]

with open('dataFeed.json', 'w') as f:
	json.dump(dataFeed, f, indent=2)
